packages/psss-server/src/connections/EntityConnection.ts

```diff
@@ -74,15 +75,52 @@ const getRelationParams = (options: RelationOptions) => {
 export class EntityConnection extends ApiConnection {
   baseUrl = ENTITY_API_URL;
 
-  fetchDescendants = async <T extends EntityOptions>(
+  public fetchCountries = async () =>
```

Comment:
I need to remember to use public/private in my classes 🤦 We should really make this a linting rule. Nice work!